packages/financial-templates-lib/test/clients/ExpiringMultiPartyEventClient.js

```diff
@@ -1,29 +1,78 @@
 const { toWei, utf8ToHex, padRight } = web3.utils;
 const winston = require("winston");
 
-const { interfaceName, parseFixed, MAX_UINT_VAL, ZERO_ADDRESS, advanceBlockAndSetTime } = require("@uma/common");
```

Comment:
changes here are again very similar to other PRs adding perp support. the main logic is multi-version testing.